Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_FrostboundBehemoth.xml

```diff
@@ -20,13 +20,13 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_FrostboundBehemoth"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>1</ArmorRating_Blunt>
+					<ArmorRating_Blunt>10</ArmorRating_Blunt>

```

Comment:
I'd leave it as-is. 

Big =/= heavily armored, and it's a giant herbivore that's "natural" defense is living where it's too damn cold for anything to come hunt it.